app/src/renderer/components/transactions/TmLiTransaction.vue

```diff
@@ -47,7 +47,10 @@ export default {
   },
   computed: {
     date() {
-      return moment(this.time).format(`MMM Do YYYY, HH:mm:ss`)
+      const time = moment(this.time)
+      return time.format(
+        `${moment().isSame(time, `day`) ? `` : `MMM Do YYYY `}HH:mm:ss`
```

Comment:
is there a test for showing the result when it's not the same day?

Reply:
As for now nothing else than a snapshot that look only at the same day version, and not the other one.
We should add a tiny test on this, I'll push it here ok?

what if we move this into app/src/renderer/scripts/time.js

Reply:
Just added the test (I had to merge develop to have Date working)